sympy/stats/frv_types.py

```diff
@@ -35,6 +37,13 @@
 'Rademacher'
 ]
 
+def _is_sym_dim(n):
+    n = sympify(n)
+    if n.atoms(Symbol, Integer) == n.atoms() and\
+        n.has(Symbol):
+        return True
+    return False
```

Comment:
Why do you need this method, can't it be done using `is_number`?

Reply:
Is `n + 1`(and similar) allowed by  `is_number`? If yes then I will replace this with `is_number`.

Reply:
You've marked this as resolved without answering the question. Why can't is_number be used?

Reply:
I think `is_number` should work. `n + 1` is not a number so it will return `False`.